### ISSUE TITLE
Region property mapper

### ISSUE THREAD

**Issue by stefanv** [scikit-image/scikit-image#1396]
It is often desired to map region properties back to the original image.  E.g., calculate the area of each region, and then fill in that value into the array (or a new array).  This would be a useful utility function, and we could potentially do it faster than the user via Cython.

**Comment by ahojnnes** [comment]
What do you mean by "fill in"?

**Comment by stefanv** [comment]
I mean creating a map of a certain property, so that when you do map[r, c]
you get the corresponding property of the image at position r, c.

**Comment by ahojnnes** [comment]
Makes sense, but let’s make this a goal for 0.12… at some point we should finally release 0.11 :-)

**Comment by vighneshbirodkar** [comment]
We can do it in a way that functions like `label2rgb` can be a subset of this more generic property mapper

**Comment by stefanv** [comment]
@vighneshbirodkar I like that tsuggestion
